src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/buildCrossTargeting/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -0,0 +1,18 @@
+<!--
```

Comment:
This file should be moved to buildMultiTargeting. Cref https://github.com/NuGet/Home/issues/4098